lib/airbrake-ruby/promise.rb

```diff
@@ -0,0 +1,75 @@
+module Airbrake
+  ##
+  # Represents a simplified promise object (similar to promises found in
+  # JavaScript), which allows chaining callbacks that are executed when the
+  # promise is either resolved or rejected.
+  #
+  # @see https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise
```

Comment:
Renamed and added 2nd link to that resource. It's no big deal, though. Concurrent-ruby aliases to `catch`.